src/smokeTest/java/uk/gov/hmcts/reform/draftstore/client/IdamClient.java

```diff
@@ -44,40 +37,30 @@ public IdamClient(
     }
 
     /**
-     * Registers a new user with Idam's testing support.
+     * Signs into Idam using credentials from config.
+     *
+     * @return Idam access token
      */
-    public void registerUser() {
-        RestAssured
-            .given()
-            .relaxedHTTPSValidation()
-            .baseUri(this.idamUrl)
-            .header(CONTENT_TYPE, APPLICATION_JSON_VALUE)
-            .body(buildUserRegistrationRequestBody())
-            .post(idamUrl + "/testing-support/accounts")
-            .then()
-            .statusCode(NO_CONTENT.value());
+    public String signIn() {
+        String authorisationCode = getAuthorisationCode();
+        return getIdamToken(authorisationCode);
     }
 
-    /**
-     * Deletes the user with Idam's testing support.
-     */
-    public void deleteUser() {
-        RestAssured
+    public String getTestToken() {
+        return RestAssured
             .given()
             .relaxedHTTPSValidation()
             .baseUri(this.idamUrl)
-            .delete("/testing-support/accounts/{email}", email)
-            .then()
-            .statusCode(isOneOf(NO_CONTENT.value(), NOT_FOUND.value()));
+            .post("/testing-support/lease?id=1&role=citizen")
```

Comment:
This endpoint is not going to be in SIDAM

Reply:
😱